ios/Podfile.lock

```diff
@@ -19,6 +19,8 @@ PODS:
     - FBSDKCoreKit (~> 5.5)
```

Comment:
Remove unwanted changes which are irrelevant to the Issue.

Reply:
done